_stbt/core.py

```diff
@@ -1143,7 +1143,8 @@ def save_frame(image, filename):
     cv2.imwrite(filename, image)
 
 
-def wait_until(callable_, timeout_secs=10, interval_secs=0, stable_secs=0):
+def wait_until(callable_, timeout_secs=10, interval_secs=0, predicate=None,
+               stable_secs=0):
```

Comment:
`predicate` should probably be the last parameter.  I like this for two reasons:

* Backwards compatibility - in case anyone's using `wait_until` without keyword arguments.
* Having callables last makes code easier to read because they can often involve brackets and it can get confusing where the parameter ends